source/gloperate/CMakeLists.txt

```diff
@@ -124,16 +124,16 @@ set(headers
     ${include_path}/stages/demos/DemoStage.h
     ${include_path}/stages/demos/DemoPipeline.h
     ${include_path}/stages/demos/ShaderDemoPipeline.h
-    ${include_path}/stages/demos/DemoAntialiasableTriangleStage.h
-    ${include_path}/stages/demos/DemoAntialiasingPipeline.h
-    ${include_path}/stages/demos/DemoDOFCubeStage.h
-    ${include_path}/stages/demos/DemoDOFPipeline.h
-    ${include_path}/stages/demos/DemoSSAORenderingStage.h
-    ${include_path}/stages/demos/DemoSSAOPostprocessingStage.h
-    ${include_path}/stages/demos/DemoSSAOPipeline.h
-    ${include_path}/stages/demos/DemoTransparencyStage.h
-    ${include_path}/stages/demos/DemoTransparencyPipeline.h
-    ${include_path}/stages/demos/DemoMultiFramePipeline.h
+#    ${include_path}/stages/demos/DemoAntialiasableTriangleStage.h
```

Comment:
It is not a preparation in the sense of "This needs to be done for #283 to work". It is to prevent a lot of changes that would be obsolete or mostly reverted after #283.
These demos need stages generating kernels. Until 38678779070fc1d4b36fedbccebebe5fdb2d832a the kernel stages were replaced by substitutes (copied from Arboretum), implemented in the gloperate module, to prevent gloperate and gloperate_glkernel from depending on each other. In the mentioned commit, other stages needed in the demos moved to gloperate_glkernel aswell, again raising the dependency problem. Writing substitutes again would leave almost no original stage left to be shown in the demos, and moving these demos over to gloperate_glkernel would become obsolete with #283.
#283 shows an easy solution to the dependency problem: The demos are extracted from the gloperate module and therefore can depend on gloperate_glkernel without forming circular dependencies.
So instead of writing a temporary workaround I chose to wait for #283 to be merged and then make the necessary adaptions.